### DISCUSSION TITLE
macOS dark theme leads to black menu text on black drop down menu background

### DISCUSSION THREAD

**Discussion by chriskuku** [FreeCAD/FreeCAD#16662] [Discussions that will turn in to issues but need refining]
OS: macOS 11.7.10
Word size of FreeCAD: 64-bit
Version: 1.0.0RC1.38642 (Git)
Build type: Release
Branch: (HEAD detached at 1.0rc1)
Hash: 60a251354e203bdb924e2190fb9f5e18b48d7362
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)

<img width="973" alt="Bildschirmfoto 2024-09-19 um 17 25 38" src="https://github.com/user-attachments/assets/b9f5ab11-df04-4588-bce3-c6bd00ac9e07">

Can this be influenced in "Preferences"?

**Comment by luzpaz** [comment]
Can you change to a different theme save and then change back...does it persist?

**Reply by chriskuku** [reply]
Thanks. Switching back and forth and leaving/entering FC again makes the dropdown menu appear with light text on dark background making it readable. But then the text fields that appear in light have white text on light background making them appear in little contrast.

<img width="758" alt="Bildschirmfoto 2024-09-22 um 11 01 00" src="https://github.com/user-attachments/assets/1390444c-7eba-4ae3-b8f0-d0a6f711cc21">

**Reply by chriskuku** [reply]
I could not make a pattern out of it but somehow it happened that the text in the menues now appears correctly in the dark theme of FC while being in dark theme under macOS. Either by toggling between the themes and/or quitting and entering FC again. 

So treat it as closed. Actually it hadn't been filed as an issue.

**Reply by luzpaz** [reply]
One of the issues seems to be this:
* https://github.com/FreeCAD/FreeCAD/issues/15131

**Comment by luzpaz** [comment]
Closing. Feel free to participate in that ticket.
